include/ppx/grfx/grfx_command.h

```diff
@@ -300,13 +300,31 @@ class CommandBuffer
         uint32_t                          setCount,
         const grfx::DescriptorSet* const* ppSets) = 0;
 
+    //
+    // Parameters count and offset are measured in DWORDs (uint32_t) aka 32-bit values.
+    // To set the first 4 32-bit values, use count = 4, offset = 0.
+    // To set the 16 DWORDs starting at offset 8, use count=16, offset = 8.
+    //
+    virtual void SetGraphicsPushConstants(
+        const grfx::PipelineInterface* pInterface,
+        uint32_t                       count,
+        const void*                    pValues,
+        uint32_t                       offset = 0) = 0;
```

Comment:
I find `offset` a bit unclear:
 - if it the offset to read from in the pValue vector? Or the push-constant offset to start updating from (like in vkCmdPushConstants).
 
 Either the param should be renamed (maybe `dstOffset` like similar cases in VK?) and/or the comment be changed to explain this bit?
 I find that `to set the 16 DWORD starting at offset` could mean we skip the first 16 words in both pValues and the push buffer.

Reply:
Changed to dstOffset.